Remove error() accidentally inserted for debugging.

OpenSSH 6.8 regression, already fixed in OpenSSH 6.9:
https://github.com/openssh/openssh-portable/commit/4d24b3b6

Remove this patch when upgrading to OpenSSH 6.9 or higher.

diff -pur old/monitor_wrap.c new/monitor_wrap.c
--- old/monitor_wrap.c
+++ new/monitor_wrap.c
@@ -153,10 +153,8 @@ mm_request_receive(int sock, Buffer *m)
 	debug3("%s entering", __func__);
 
 	if (atomicio(read, sock, buf, sizeof(buf)) != sizeof(buf)) {
-		if (errno == EPIPE) {
-			error("%s: socket closed", __func__);
+		if (errno == EPIPE)
 			cleanup_exit(255);
-		}
 		fatal("%s: read: %s", __func__, strerror(errno));
 	}
 	msg_len = get_u32(buf);
